tests/dragonfly/instance.py

```diff
@@ -425,10 +425,6 @@ def create(self, existing_port=None, path=None, version=100, **kwargs) -> DflyIn
             args.setdefault("list_experimental_v2")
         args.setdefault("log_dir", self.params.log_dir)
 
-        if version >= 1.21 and "serialization_max_chunk_size" not in args:
-            # Add 1 byte limit for big values
-            args.setdefault("serialization_max_chunk_size", 1)
```

Comment:
oh but I do want to have serialization_max_chunk_size for all tests, not just test_replication_all
Just use default of 1024 instead of 1 f.e

Reply:
I was about to write we do but then I realized I defaulted it to 32_MB :rofl: so yes we don't :facepalm: Will fix it!